packages/eventual-send/rollup.config.js

```diff
@@ -6,9 +6,9 @@ export default [
         file: 'dist/eventual-send.umd.js',
         format: 'umd',
         name: 'EventualSend',
-        globals: {
-          '@agoric/harden': 'harden',
-        },
+        // globals: {
```

Comment:
@michaelfig can we delete this config file now? I think I got rid of the bundling step, but missed the config file.

Reply:
Yes, that can be deleted.